packages/nvidia-k8s-device-plugin/1001-Ensure-that-generated-CDI-specs-do-not-contain-enabl.patch

```diff
@@ -0,0 +1,36 @@
+From 49dc098d513923505a66bc8380ed80c0ce9a0165 Mon Sep 17 00:00:00 2001
+From: Evan Lezar <elezar@nvidia.com>
+Date: Mon, 10 Mar 2025 14:44:45 +0200
+Subject: [PATCH] Ensure that generated CDI specs do not contain
+ enable-cuda-compat hooks
+
+Signed-off-by: Evan Lezar <elezar@nvidia.com>
```

Comment:
Patch is missing the `cherry picked from commit`, could you please generate the patch with `git cherry-pick -x` so we keep track of where it came from?